tests/routes/visitor_name.test.js

```diff
@@ -45,9 +46,10 @@ test('POST /api/user/name-from-scan | non-viable hash ', async t => {
     .post('/api/user/name-from-scan')
     .set('authorization', token)
     .send(failPayload)
```

Comment:
This shouldn’t be expecting a 200 if something went wrong. Can you update the status code in the route?